akka-http/src/main/scala/akka/http/scaladsl/server/directives/FileAndResourceDirectives.scala

```diff
@@ -186,6 +181,11 @@ trait FileAndResourceDirectives {
 }
 
 object FileAndResourceDirectives extends FileAndResourceDirectives {
+  private val withRangeSupportAndPrecompressedMediaTypeSupportAndExtractSettings =
```

Comment:
You *always* look at the source to see what it's doing!
But now, for this method, you'll never have to again because the name is so descriptive... :)